bom/pom.xml

```diff
@@ -41,10 +41,11 @@
 
         <!-- Hibernate ORM -->
         <hibernateVersion>5.2.13.Final</hibernateVersion>
+        <hibernate-orm.module.slot>5.2</hibernate-orm.module.slot>
```

Comment:
I'm not sure it makes sense to declare them in the bom. I'm not sure it's worth it to make them visible to the users using our BOM.

What's the rationale for this change?

Reply:
1# consistency: the Search one is already there.
2# It helps to have the module slot property close the the full version.
3# I see no problem in having these in the BOM, it doesn't hurt and it might be useful to some.

Reply:
Ah missed that the Search one was already there. Would you mind fixing the case to be consistent with the rest of the properties? I know the Search one was already wrong but we should better fix it once and for all.

I consider the BOM as being exposed to the users so it would be nice if we could be consistent.

Thanks!